python/node2vec_online.py

```diff
@@ -0,0 +1,166 @@
+import time
+import multiprocessing
+from mage.node2vec_online_module import GensimWord2Vec, StreamWalkUpdater
+
+MAX_THREADS = multiprocessing.cpu_count()
+
+
+class Node2VecContext:
+    def __init__(self):
+        self._updater = None
+        self._learner = None
+        self._start_time = None
+
+    @property
+    def updater(self):
+        return self._updater
+
+    @property
+    def learner(self):
+        return self._learner
+
+    @property
+    def start_time(self):
+        return self._start_time
+
+    @property
+    def elapsed_time(self) -> int:
+        return time.time() - self._start_time
+
+    @updater.setter
+    def updater(self, updater):
+        self._updater = updater
+
+    @learner.setter
+    def learner(self, learner):
+        self._learner = learner
+
+    @start_time.setter
+    def start_time(self, start_time):
+        self._start_time = start_time
+
+    def is_initialized(self):
+        return self._learner is None or self._updater is None or self._start_time is None
```

Comment:
`is_initialized` is `true` once some of attributes is None, is this expected behaviour?